cpp/include/cudf/detail/hashing.hpp

```diff
@@ -66,15 +87,15 @@ std::unique_ptr<column> serial_murmur_hash3_32(
  * Adapted from Boost hash_combine function, modified for 64-bit
  * https://www.boost.org/doc/libs/1_35_0/doc/html/boost/hash_combine_id241013.html
  *
- * @param lhs The first hashed value
- * @param rhs The second hashed value
- * @return Combined hash value
+ * @param lhs The first hash value.
+ * @param rhs The second hash value.
+ * @return Combined hash value.
```

Comment:
The smallest nitpick possible - I thought we don't end @param/@return comments with periods.

Reply:
I have been making this mistake a lot, if that's the case. I'm planning to give a Better Code talk about writing Doxygen docstrings in the future, so I'd appreciate a reference on this if you have one.
```suggestion
 * @param lhs The first hash value
 * @param rhs The second hash value
 * @return Combined hash value
```

Reply:
My comment is based on https://github.com/rapidsai/cudf/blob/branch-22.04/cpp/docs/DOCUMENTATION.md

Reply:
Thank you! It's almost too easy to point to the developer docs. 😅